### ISSUE TITLE
Panicked result on exa -l in certain directories

### ISSUE THREAD

**Issue by patrickn** [ogham/exa#592]
exa v0.9.0

```
~/Sites/public> RUST_BACKTRACE=1 exa -l
thread 'thread '<unnamed><unnamed>' panicked at '' panicked at 'called `Result::unwrap()` on an `Err` value: SystemTimeError(6795364577.576378s)called `Result::unwrap()` on an `Err` value: SystemTimeError(6795364577.576378s)', ', src/libcore/result.rssrc/libcore/result.rs::999999::55

thread '<unnamed>' panicked at 'called `Result::unwrap()` on an `Err` value: SystemTimeError(6795364577.576378s)', thread 'src/libcore/result.rs<unnamed>:' panicked at '999called `Result::unwrap()` on an `Err` value: SystemTimeError(6795364577.576378s):', 5src/libcore/result.rs
:999:5
note: Some details are omitted, run with `RUST_BACKTRACE=full` for a verbose backtrace.
stack backtrace:
   0: std::sys::unix::backtrace::tracing::imp::unwind_backtrace
   1: std::panicking::default_hook::{{closure}}
   2: std::panicking::rust_panic_with_hook
   3: std::panicking::continue_panic_fmt
   4: rust_begin_unwind
   5: core::panicking::panic_fmt
   6: core::result::unwrap_failed
   7: exa::output::table::Table::display
   8: <F as scoped_threadpool::FnBox>::call_box
zsh: abort      RUST_BACKTRACE=1 exa -l
```

This is the directory listing when using 'ls -l':

```
~/Sites/public> ls -l
total 112
-rw-r--r--   1 pneave  staff   4956 Oct  1 10:39 404.html
-rw-r--r--   1 pneave  staff    404 Sep  8 12:11 _redirects
drwxr-xr-x   5 pneave  staff    160 Oct  1 10:39 categories
drwxr-xr-x   3 pneave  staff     96 Oct  1 10:39 contact
drwxr-xr-x   3 pneave  staff     96 Aug 30  1754 css
drwxr-xr-x   5 pneave  staff    160 Oct  1 10:45 cv
-rw-r--r--   1 pneave  staff     53 Mar 17  2019 googlee5e7627938d01209.html
-rw-r--r--   1 pneave  staff    859 Mar 17  2019 humans.txt
drwxr-xr-x  15 pneave  staff    480 Aug 30  1754 img
-rw-r--r--   1 pneave  staff   5415 Oct  1 10:39 index.html
-rw-r--r--   1 pneave  staff   1311 Oct  1 10:39 index.xml
drwxr-xr-x   5 pneave  staff    160 Aug 30  1754 js
drwxr-xr-x   3 pneave  staff     96 Aug 30  1754 pgp
-rw-r--r--   1 pneave  staff  10187 Mar 17  2019 privacy.txt
-rw-r--r--   1 pneave  staff     28 Oct  1 10:39 robots.txt
-rw-r--r--   1 pneave  staff     61 Mar 17  2019 security.txt
-rw-r--r--   1 pneave  staff    502 Oct  1 10:39 sitemap.xml
drwxr-xr-x   5 pneave  staff    160 Oct  1 10:39 tags
```

**Comment by naiveai** [comment]
This depends so heavily on the state of your system that I'm not sure if it's reproducible at all.

**Comment by flypenguin** [comment]
Same with me.

Diagnosis: 

- git-managed directory
- It happens when I try to "exa -l" the `public/` folder.
- This folder is in my `.gitignore` as `/public/`.
- This could not be reproduced in a different git-managed directory

```text
$ RUST_BACKTRACE=full ll public
thread '<unnamed>' panicked at 'called `Result::unwrap()` on an `Err` value: SystemTimeError(6795364578.871345152s)', src/libcore/result.rs:1084:5thread '
<unnamed>stack backtrace:
thread '' panicked at '<unnamed>called `Result::unwrap()` on an `Err` value: SystemTimeError(6795364578.871345152s)' panicked at 'thread '', called `Result::unwrap()` on an `Err` value: SystemTimeError(6795364578.871345152s)<unnamed>src/libcore/result.rs', ' panicked at ':src/libcore/result.rscalled `Result::unwrap()` on an `Err` value: SystemTimeError(6795364578.871345152s)1084:', :1084src/libcore/result.rs5::
thread '51084<unnamed>
:' panicked at '5called `Result::unwrap()` on an `Err` value: SystemTimeError(6795364578.871345152s)
', src/libcore/result.rsthread ':1084<unnamed>:5' panicked at '
called `Result::unwrap()` on an `Err` value: SystemTimeError(6795364578.871345152s)', src/libcore/result.rs:1084:5
   0:        0x105c6eced - std::panicking::default_hook::{{closure}}::hffc69ca72fa98b80
   1:        0x105c6e7bc - std::panicking::rust_panic_with_hook::h058ef423beec40ce
   2:        0x105c6ef7d - std::panicking::continue_panic_fmt::h0bb2cdb45a553955
   3:        0x105c6f249 - rust_begin_unwind
   4:        0x105d00dfa - core::panicking::panic_fmt::h2413d97d26203710
   5:        0x105d01399 - core::result::unwrap_failed::hef33208e3f23e07a
   6:        0x105c346bd - exa::output::table::Table::display::h7d61868c2e1aa281
   7:        0x105c47955 - <F as scoped_threadpool::FnBox>::call_box::hfeff2cdcb76e7a57
   8:        0x105c5fd6e - std::sys_common::backtrace::__rust_begin_short_backtrace::hf388fc36537a8fe9
   9:        0x105c6089e - core::ops::function::FnOnce::call_once{{vtable.shim}}::hd49318dcebc5ef76
  10:        0x105c7913d - <alloc::boxed::Box<F> as core::ops::function::FnOnce<A>>::call_once::hf778f932793a5a82
  11:        0x105c79767 - std::sys::unix::thread::Thread::new::thread_start::ha4ccb7099fa6037f
  12:     0x7fff6e733e65 - _pthread_start
stack backtrace:
   0:        0x105c6eced[1]    47891 abort      RUST_BACKTRACE=full exa -lFh public

**Comment by ogham** [comment]
The fix for this (#670) has been merged and will go out with the next release.
